scripts/west_commands/blobs.py

```diff
@@ -151,6 +153,23 @@ def fetch(self, args):
                 self.dbg('Blob {module}: {abspath} is up to date'.format(**blob))
                 continue
             self.inf('Fetching blob {module}: {abspath}'.format(**blob))
+
+            if blob['license-click-through']:
+                user_input = input("For this blob, need to read and accept license to continue."
+                                   "Read it? [y/n]: ")
+                if user_input.upper() != "Y":
+                    self.inf('Warn: skip fetching this blob.')
```

Comment:
should we only test for `Y` ?
Or should we actually test specifically `Y` and `N` and request user to use one of those, so that typing `yes` will re-request the user to type `y/n`.

Right now, if a user types `yes<enter>`, then the west command will continue as if user typed `N`.

Reply:
Will request user to input y or n, and check strictly.
Thanks.